src/container-mapping.ts

```diff
@@ -0,0 +1,184 @@
+import { CommandType, Constants, getEncodedContent, writeToOutStream } from "./msdo-helpers";
+import { IMicrosoftSecurityDevOps } from "./msdo-interface";
+import * as https from "https";
+import * as core from '@actions/core';
+import * as exec from '@actions/exec';
+
+const sendReportRetryCount: number = 1;
+
+/**
+ * Represents the tasks for container mapping that are used to fetch Docker images pushed in a job run.
+ */
+export class ContainerMapping implements IMicrosoftSecurityDevOps {
+    readonly succeedOnError: boolean;
+
+    constructor() {
+        this.succeedOnError = true;

```

Comment:
Given that this task will be added manually by the customer, we should not really succeed on Error and surface it back to the customer.